hieradata/role/amor.yaml

```diff
@@ -7,3 +7,4 @@ classes:
   - "profile::core::docker::prune"
   - "profile::core::nfsclient"
   - "profile::ts::dco"
+  - "profile::ts::love"
```

Comment:
Please add spec coverage for this change to `spec/hosts/roles/amor_spec.rb`.

Reply:
added